sdcm/nemesis.py

```diff
@@ -629,8 +629,8 @@ def _modify_table_property(self, name, val):
         disruption_name = "".join([p.strip().capitalize() for p in name.split("_")])
         self._set_current_disruption('ModifyTableProperties%s %s' % (disruption_name, self.target_node))
 
-        ks_cfs = self.loaders.get_non_system_ks_cf_list(loader_node=random.choice(self.loaders.nodes),
-                                                        db_node_ip=self.target_node.ip_address)
+        ks_cfs = self.loaders.get_non_system_non_counter_ks_cf_list(loader_node=random.choice(self.loaders.nodes),
```

Comment:
As mentioned I think by @bentsi, can we filter the counter, only for the specific problematic case ? cause this change would stop doing any of the alter changes on counter tables... and we'll miss any issues related to alter on counters tables

Reply:
You are right, currently I only touch problem in ALTER default_time_to_live

Reply:
@amoskong please improve your PR to be specific for ALTER default_time_to_live

Reply:
> @amoskong please improve your PR to be specific for ALTER default_time_to_live

It's PR from @juliayakovlev  ;-)

Reply:
Fixed